frontend/src/entry/model/Files.tsx

```diff
@@ -14,13 +16,58 @@ type FilesProps = {
 
 export default function Files({ model }: FilesProps) {
   const { entryFiles, isEntryFilesLoading, isEntryFilesError } = useGetModelFiles(model.id)
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null)
+  const menuOpen = Boolean(anchorEl)
+  const [orderByValue, setOrderByValue] = useState('createdAt')
+  const [orderByButtonTitle, setOrderByButtonTitle] = useState('Order by')
+  const [ASCorDESC, setASCorDESC] = useState('DESC')
+
+  const sortFilesByValue = useMemo(
+    () => (a: FileInterface, b: FileInterface) => {
+      if (ASCorDESC === 'DESC') {
+        return a[orderByValue] < b[orderByValue] ? -1 : 1
+      }
+      return a[orderByValue] > b[orderByValue] ? -1 : 1
+    },
+    [ASCorDESC, orderByValue],
+  )
+
+  const checkMenuOption = useCallback(
+    (menuOption: string) => {
+      if (menuOption === orderByValue) {
+        return <Check sx={{ width: '100%' }} color='primary' />
+      } else {
+        return <Check sx={{ width: '100%' }} color='primary' opacity={0} />
+      }
+    },
+    [orderByValue],
+  )
+
+  const selectedMenuOption = useCallback(
+    (menuOption: string) => {
+      if (menuOption === orderByValue) {
+        return true
+      } else {
+        return false
+      }
+    },
+    [orderByValue],
+  )
+
+  function handleMenuButtonClick(event: React.MouseEvent<HTMLButtonElement>) {
```

Comment:
Change to 

`(event: MouseEvent<HTMLButtonElement>)`

And import `MouseEvent` from `react`